src/ol/layer/WebGLPoints.js

```diff
@@ -72,24 +71,24 @@ class WebGLPointsLayer extends Layer {
     super(baseOptions);
 
     /**
-     * @type {import('../style/LiteralStyle.js').LiteralStyle}
+     * @private
+     * @type {import('../webgl/ShaderBuilder.js').StyleParseResult}
      */
-    this.style = options.style;
-
-    assert(this.style.symbol !== undefined, 65);
+    this.parseResult_ = parseSymbolStyle(
+      /** @type {import('../style/LiteralStyle.js').LiteralStyle} */
```

Comment:
This typecast is not necessary: typescript already knows the type of `options.style`. can you please change to:
```js
this.parseResult_ = parseSymbolStyle(options.style.symbol);
```